map-widget/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+# map widget to show locations
+
+# library used by mobile build
+set(SUBSURFACE_MAPWIDGET_SRCS
+	qmlmapwidgethelper.cpp
+)
```

Comment:
This is truly just a question... but... is it worth adding this for one file? In other words, besides logical consistency and what not, what do we gain by doing this?

Reply:
That argument is valid for all CMakeList.txt files in subdirectories. Why not just make one big central file (e.g. mobile-widgets, contained a lot of entries in the central file).

We do not gain performance or anything, but consistency means easier to understand and thereby easier to maintain (at least for me). The other gain is that each root directory is a library.

And I "believe" that in a short future, there will be a PR with a dynamic map that allows the user to change locations (at least the current experiments looks good).